arch/riscv/sbi/sbi_emulate_csr.c

```diff
@@ -90,6 +90,9 @@ int sbi_emulate_csr_read(int csr_num, u32 hartid, ulong mstatus,
 	case CSR_MHARTID:
 		*csr_val = hartid;
 		break;
+	case CSR_MIMPID:
+		*csr_val = csr_read(CSR_MIMPID);
+		break;
 	default:
```

Comment:
Shall we add CONFIG_SBI_CSR_MIMPID to opt-out this code when it is not necessary?

Reply:
done